test/templates/toggle/component.njk

```diff
@@ -8,8 +8,10 @@ toggle:
 -->
 {% macro toggle(properties={}, modifier={}, state={}, aria={}, classes=[]) -%}
 {%- set _id = properties.id or utils.id() -%}
-<span class="fd-toggle{{ modifier.block | modifier('toggle') }}{{ classes | classes }}"{{ aria | aria }}>
-    <input type="checkbox" name="" value="" id="{{ _id }}"{{ ' disabled' if state.disabled }}{{ ' checked' if state.checked }}>
-    <span class="fd-toggle__switch" role="presentation"></span>
-</span>
+<label>
```

Comment:
This `label` cannot be here. It will be output as a duplicate when used in a `form__group`.

Reply:
If I don't add the label I don't have a way to test it in localhost:3030. Open to ideas :)